README.md

```diff
@@ -16,41 +16,7 @@ None
 
 [//]: # (addons)
 
-Available addons
-----------------
-addon | version | maintainers | summary
---- | --- | --- | ---
-[barcode_action](barcode_action/) | 14.0.1.0.1 |  | Allows to use barcodes as a launcher
-[base_cancel_confirm](base_cancel_confirm/) | 14.0.1.0.3 | [![kittiu](https://github.com/kittiu.png?size=30px)](https://github.com/kittiu) | Base Cancel Confirm
```

Comment:
Can you exclude the change in the readme? It is a bit of a useless diff because it will be written again.

Regarding the module change, LGTM